### ISSUE TITLE
Create custom commands

### ISSUE THREAD

**Issue by damjanaleksiev** [tiff/wysihtml5#348]
I plan to use the editor for a project the will probably require a few more commands than the preset ones, but I have not found any reference for generating new ones in the wiki.

So can anyone please tell me, is it possible, and if yes, how to extend wysihtml5 with custom commands?  

Thanks.

**Comment by CurtisSV** [comment]
I agree that this would be useful. I'm trying to figure it out myself. Have you had any luck?

**Comment by CurtisSV** [comment]
If you're using the "advanced Demo", click view source and you'll see that the page depends on two .js files. One of the files is "wysihtml5-0.4.0.pre.js" 

Open that file and search for "ENTER_KEY". Start playing around and you should be able to figure it out.

**Comment by damjanaleksiev** [comment]
Thanks for your response CurtisSV, Its good to know that the feature exists, I'll definitely check it out.

**Comment by damjanaleksiev** [comment]
I have seen the functionality you were referring to, but it seems it is about keyboard event detection. I was talking commands in the general sense, such as for example adding the functionality to change fonts in the editor.
Anyone has a tip for this?

**Comment by damjanaleksiev** [comment]
I manged to create a custom command. You basically need to create a function similar to the one line 6856 (witch handles the bold command). Its not too difficult to do, most of the stuff is done by the plugin anyway. You basically need just change the name and define a few classes in the css file.
